demos/demo-jndi-webapp/src/main/config/modules/demo-jndi.mod

```diff
@@ -17,5 +17,4 @@ demo-mock-resources
 [files]
 basehome:modules/demo.d/demo-jndi.xml|webapps/demo-jndi.xml
 maven://org.eclipse.jetty.demos/demo-jndi-webapp/${jetty.version}/war|webapps/demo-jndi.war
-maven://jakarta.mail/jakarta.mail-api/2.0.0-RC4/jar|lib/ext/jakarta.mail-api-2.0.0-RC4.jar
-maven://jakarta.transaction/jakarta.transaction-api/2.0.0-RC1/jar|lib/ext/jakarta.transaction-api-2.0.0-RC1.jar
+maven://jakarta.mail/jakarta.mail-api/2.0.0/jar|lib/ext/jakarta.mail-api-2.0.0.jar
```

Comment:
Where did `jakarta.transaction` go?

Reply:
Ah. I understand now.
Commit 3ae3e1fa47a3a226497e2be71c2b6ef56b851f49 helped.

Reply:
It should never have been put there, as that jar is in $jetty-home/lib and is put onto the classpath by `.plus.mod`